starter-code/apple-pie-starter-code/images/main.css

```diff
@@ -0,0 +1,120 @@
+/* ---- reset ---- */
+
+html {
+    box-sizing: border-box;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  
+  body {
+      margin: 0px;
+      padding: 0px;
+  }
+  
+  
+  
+  /* ---- typography ---- */
+  
+  body {
+      font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif
+  }
+  
+  h1 {}
+  
+  h2 {}
+  
+  h3 {
+      font-weight: 600;
+      font-size: 25px;
+  }
+  
+  
+  
+  
+  /* ---- layout ---- */
+  
+  #site-header {}
+  
+  #site-footer {}
+  
+  #site-main {}
+  
+ /* #everything-else {
+      background: white;
+  } */
+
+  #title {
+      padding-top: 0px;
+      margin-top: -12%;
+      width: 100%;
+      height: 830px;
+      display: block;
+      text-align: center;
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-image: url("./apple-pie.jpg")
```

Comment:
Instead of adding the css file to image folder try to use ../ (notice the two dots) to leave the folder you are. It will look like this:
```
background-image: url("../images/apple-pie.jpg")
```